.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,18 +34,15 @@ See [USWDS release notes](https://github.com/uswds/uswds/releases) for examples.
 -->
 
 ## Breaking change
-_Indicate if this update is a breaking change._
-<!--
-Use clear language to:
-1. Indicate if this update is a breaking change.
-2. Explain what actions are required for the user to remediate the break.
-   For clarity, include a statement here even if there is no breaking change:
-   "This is not a breaking change."
-   ":warning: This is potentially a breaking change."
-   ":warning: This is a breaking change."
--->
+
+_Indicate if this update is a breaking change with one of the following statements:_
```

Comment:
@mejiaj I took a stab at including some visible short, clear instructions so that it can more closely match the format in the other sections. Let me know what you think!

Reply:
Glad you like the suggestions, this looks good. We might want to consider making the `one` bold to make it standout a little more.

```suggestion
_Indicate if this update is a breaking change with **one** of the following statements:_
```